swift.html.markdown

```diff
@@ -104,10 +105,13 @@ if let someOptionalStringConstant = someOptionalString {
 }
 
 // Swift has support for storing a value of any type.
-// AnyObject == id
-// Unlike Objective-C `id`, AnyObject works with any value (Class, Int, struct, etc.)
-var anyObjectVar: AnyObject = 7
-anyObjectVar = "Changed value to a string, not good practice, but possible."
+// Any == id
+// Unlike Objective-C `id`, Any works with any value (Class, Int, struct, etc.)
+var anyVar: Any = 7
+anyVar = "Changed value to a string, not good practice, but possible."
+
+// Also there is similar type – `AnyObject`, which works only for objects
```

Comment:
I'll rewrite this. I mean 
`AnyObject` == `id` from Objective-C
`Any` – also works with any scalar values (Class, Int, struct, etc.)
And in current version of Swift we can't do such things `var anyObjectVar: AnyObject = 7`